api/src/routers/channel-info/index.ts

```diff
@@ -47,7 +47,31 @@ export const channelInfoRouter = Router();
  *       in: query
  *       required: false
  *       schema: 
- *         $ref: "#/components/schemas/ChannelInfoSearchSchema"
+ *         type: object
```

Comment:
Why did you remove the ChannelInfoSearchSchema schema here?

Reply:
In the query the keys are written with dashes, in the schema the keys are in camelCase so the keys in the query are not compatible with the ones in the schema